File: 1-easy/problems/2144-minimum-cost-of-buying-candies-with-discount/solution.ts
function minimumCost(cost: any): boolean | number | string | any {
  """
  :type cost: List[int]
  :rtype: int
  """
  cost.sort(reverse=true)
  return sum(x for i, x in enumerate(cost) if i%3 != 2);
}

export { minimumCost };

File: 1-easy/problems/2144-minimum-cost-of-buying-candies-with-discount/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def minimumCost(self, cost):
        """
        :type cost: List[int]
        :rtype: int
        """
        cost.sort(reverse=True)
        return sum(x for i, x in enumerate(cost) if i%3 != 2)
